STRATEGIC-INITIATIVES.md

```diff
@@ -0,0 +1,46 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  With the less-tangible approach to the work that the Community 
+Committee does, having a way to track this work is important.
+
+For each strategic initiative, the Community Committee's goal is to have a
+single point of reference - a champion - that understands what's happening
+with the initiative to the fullest extent posssible, and can report on the
+initiative as needed.
+
+A review of the initiatives will be a standing item on the Community Committee agenda (even
```

Comment:
this is kind of bringing up a question - if there are a lot of initiatives, having them as a constant part of our agenda slims down the flexible parts of our agenda, e.g. leaves us less time to discuss current issues (assuming this is reported on during our meetings). is this intentional?

Reply:
Not intentional, but this does have precedent in the TSC. I don't think there's been an issue there, but maybe @mhdawson could shed some light on that. In the times I have tuned in, if any initiatives have reported anything other than "No update - everything's going well" there've only been one or two. Generally they've not been lengthy, either - just a check in to see if there's anything the Committee can do to help, or if the Committee has anything it needs to take care of for the initiative.

Does that make sense and/or do you see a possible issue? Happy to discuss as necessary.

Reply:
nope, that makes sense

Reply:
It has not been an issue so far.  I think its also fair to skip/limit that section if you have a very full agenda of other things that need to be addressed.